server/test/serverTest.js

```diff
@@ -1,575 +1,661 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
If you revert all the changes in the test files, do they still pass?

It's a good thing to try. Because right now, we have confidence in our tests to check against changes in non-test code. So running them as they were before against these ESLint/Prettier changes will mean that we haven't broken any of the things that we have tests for.

You could also try it the other way around and revert all the non-test code and make sure that the tests still all pass (make sure to check the counts). This will give you more confidence that the ESLint/Prettier changes made to the test code hasn't broken the tests.

Reply:
I was re-running tests at every step but I can do it more?

Reply:
That might be OK. It depends on whether any step changed _both_ the test code and the non-test code simultaneously. Because if it changed both at the same time, then I have less confidence that they are still checking what they were checking before. Does that make sense?

Reply:
I also did the changes on the tests last I believe aside from the auto-fixable ones

Reply:
But I can just copy in what was there from master and run them, not a labour-intensive thing 😅

Reply:
> That might be OK. It depends on whether any step changed _both_ the test code and the non-test code simultaneously. Because if it changed both at the same time, then I have less confidence that they are still checking what they were checking before. Does that make sense?

oops sorry this comment hadn't loaded but yes it does make sense! nothing auto-fixable should have done that in any case, I was generally only running into suddenly failing tests when incrementally doing manual changes like '==' vs '==='

Reply:
but I will check!

Reply:
This passes when reverted to what's on `master` :)